### PR TITLE
ci/cd: added pr-comment-bot.yml

### PR DESCRIPTION
**Pull request by bang9**
## External Contributions

This project is not yet set up to accept pull requests from external contributors.

If you have a pull request that you believe should be accepted, please contact
the Developer Relations team <developer-advocates@sendbird.com> with details
and we'll evaluate if we can setup a [CLA](https://en.wikipedia.org/wiki/Contributor_License_Agreement) to allow for the contribution.

### Description Of Changes

- Added comment bot workflow 
*Should add the bot's `CIRCLECI_API_TOKEN` to the secret. If you give me permission, I'll add it.

### CODE REVIEW

**Review by sravan-s — APPROVED** [review]
Thanks 👍  Can we merge on monday pls? I will give you access by then, also we are doing some other things

**Comment by bang9** [comment]
> Thanks 👍 Can we merge on monday pls? I will give you access by then, also we are doing some other things

@sravan-s yes yes, no problem

File: .github/workflows/pr-comment-bot.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: PR comment bot
+on:
+  issue_comment:
+    types: [created]
+jobs:
+  pr-comment:
```
Comment (sravan-s) [inline .github/workflows/pr-comment-bot.yml:6]: https://github.com/sendbird/release-automation-action#commands 
good to have this explicit ~ as comment

Comment (sravan-s) [reply .github/workflows/pr-comment-bot.yml:6]: cc @bang9

Comment (bang9) [reply .github/workflows/pr-comment-bot.yml:6]: Could you please explain in a bit more detail?

File: .github/workflows/pr-comment-bot.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: sendbird/release-automation-action@0.0.3
```
Comment (sravan-s) [inline .github/workflows/pr-comment-bot.yml:10]:
```suggestion
      # see list of commands: https://github.com/sendbird/release-automation-action#commands
      - uses: sendbird/release-automation-action@0.0.3
```

**Pull request merged**